gobblin-runtime/build.gradle

```diff
@@ -103,6 +103,14 @@ artifacts {
 test {
     workingDir rootProject.rootDir
     maxParallelForks = 1
+  /*
```

Comment:
Is there a reason to leave these commented lines here? If not, please remove them.